From fbacaeeb856ddf2de1e87a85ec8e421ad2fe9695 Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Tue, 23 Jul 2019 08:29:33 +0700
Subject: [PATCH 049/433] ARM: dts r8a77470: Change power-domain of DMAC

Change power-domain from cpg-clock to SYSC.

Instantiate the two system DMA controllers in the r8a77470 device tree.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index cbc5b99..f29461e 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -444,7 +444,7 @@
 					  "ch12", "ch13", "ch14";
 			clocks = <&mstp2_clks R8A77470_CLK_SYS_DMAC0>;
 			clock-names = "fck";
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			#dma-cells = <1>;
 			dma-channels = <15>;
 		};
@@ -476,7 +476,7 @@
 					  "ch12", "ch13", "ch14";
 			clocks = <&mstp2_clks R8A77470_CLK_SYS_DMAC1>;
 			clock-names = "fck";
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			#dma-cells = <1>;
 			dma-channels = <15>;
 		};
-- 
2.7.4

